### ISSUE TITLE
BUG: Docker cannot export .env anymore

### ISSUE THREAD

**Issue by whyper-mateus** [Infisical/infisical#201]
https://github.com/Infisical/infisical/blob/68b99b9f007b1c654664e99f088c181aeeff25d7/cli/packages/cmd/export.go#L32

This line was changed and anytime we try to run infisical export we get this following message:
```
You must be logged in to run this command. To login, run [infisical login]
```

We use infisical export to cache secrets to reduce the startup time, but with this update, this command cannot recognize the infisical token.

**Comment by maidul98** [comment]
We have recently made several updates to the CLI and its underlying API and the restriction was added unintentionally during these updates. I will let you know when this gets resolved!

**Comment by maidul98** [comment]
This has been patched in 0.2.1! Please update your CLI and let us know if you face any other issues
